src/sentry/api/endpoints/organization_health.py

```diff
@@ -250,11 +269,15 @@ def get(self, request, organization):
         if not project_ids:
             return self.empty()
 
-        environment = self.get_environment(request, organization)
+        environments = self.get_environments(request, organization)
         query_condition = self.get_query_condition(request, organization)
 
         rollup = int(interval.total_seconds())
 
+        filter_keys = {'project_id': project_ids}
+        if environments:
+            filter_keys['environment'] = environments
```

Comment:
Is `environment` a top level filter now? Before it was a part of tags and needed to be in conditions.

Reply:
do you mean in snuba? There is an environment column in snuba and this comment also suggests environment can be used as in filter_keys: https://github.com/getsentry/sentry/blob/master/src/sentry/utils/snuba.py#L72

Reply:
Yeah, so looking in snuba code, this ends up being identical to `conditions`. So not sure why it's being changed into `filter_keys`, but it ends up the same place since Snuba itself doesn't support `filter_keys`, and they're just translated from `filter_keys` into an `[environment, IN, ...]` condition anyways, so this is fine.

My concern is `environment` vs `tags[environment]`. There's also the behavior where I check for `IS NULL` on the "no environment" environment. Is there any idea how that's handled through this API? Is it possible to select "no environment" environment and another environment with a name?